sympy/vector/coordsysrect.py

```diff
@@ -173,6 +174,73 @@ def __str__(self, printer=None):
     def __iter__(self):
         return iter([self.i, self.j, self.k])
 
+    def connect_to_standard_cartesian(self,  curv_coord_type):
+        """
+        Change the type of orthogonal curvilinear system. It could be done
+        by tuple of transformation equations or by choosing one of pre-defined
+        coordinate system.
+
+        Parameters
+        ==========
+
+        :param curv_coord_type: str, tuple
+
+        """
+        if type(curv_coord_type) is str:
+            self._set_transformation_equations_mapping(curv_coord_type)
+            self._set_lame_coefficient_mapping(curv_coord_type)
+
+        elif type(curv_coord_type) is tuple and len(curv_coord_type) == 3:
+            self._transformation_eqs = curv_coord_type
+            self._h1, self._h2, self._h3 = self._calculate_lame_coefficients(curv_coord_type)
+        elif type(curv_coord_type) is tuple and len(curv_coord_type) == 2:
```

Comment:
`2` means that tuple contains two tuple:
```python
connect_to_standard_cartesian((x, y, z), (x, y, z))
```
I think, we should figure out something better.